documentation/modules/new-migrating-virtual-machines-cli.adoc

```diff
@@ -865,28 +865,34 @@ The template follows the Go template syntax and has access to the following vari
 * `"{{if .Shared}}shared-{{end}}{{.VmName}}-{{.DiskIndex}}"`
 +
 <11> Optional:
-* When set to `true`, {project-short} adds one or more randonly generated alphanumeric characters to the name of the PVC in order to ensure all PVCs have unique names.
-* When set to `false`, if you specify a `pvcNameTemplate`, {project-short} does not add such charchters to the name of the PVC.
+* When set to `true`, {project-short} adds one or more randomly generated alphanumeric characters to the name of the PVC in order to ensure all PVCs have unique names.
+* When set to `false`, if you specify a `pvcNameTemplate`, {project-short} does not add such characters to the name of the PVC.
 +
 include::snip_pvcNameTemplateUseGenerateName-warn.adoc[]
 +
-<12> [[callout12]]Optional: Specify a template for the volume interface name for the VMs in your plan.
+<12> Determines whether VMs are converted before migration using a tool named `virt-v2v` that makes the VMs compatible with {virt}.
```

Comment:
```suggestion
<12> Determines whether VMs are converted before migration using the `virt-v2v` tool, that makes the VMs compatible with {virt}.
```

a. maybe add a link to virt-v2v tool docs ?
https://access.redhat.com/articles/1351473

b. i don't know english, so i may be totally wrong here, but a `a cat named Mitzi` template, sound strange to my none english speaking ear.

Reply:
i am missing the https://issues.redhat.com/browse/MTV-3009 documentation for `useCompatibilityMode` when skipping vm conversion we can also control the compatibility mode, e.g. skip conversion but set compatibility mode on, will make the vm work even if it's not concerted (set as default when conversion is off)